src/datatype.c

```diff
@@ -935,6 +935,10 @@ JL_DLLEXPORT jl_datatype_t *jl_new_primitivetype(jl_value_t *name, jl_module_t *
                                         jl_emptysvec, jl_emptysvec, jl_emptysvec, 0, 0, 0);
     uint32_t nbytes = (nbits + 7) / 8;
     uint32_t alignm = next_power_of_two(nbytes);
+# if defined(_CPU_X86_) && !defined(_OS_WINDOWS_)
+    if (alignm == 8)
```

Comment:
Is this correct? https://godbolt.org/z/ec3bbo69T seems to show that it's 8 byte aligned

Reply:
The godbolt link is for windows right? and this code is for not windows.

Reply:
Oh I missed the `!`, you're right